.github/actions/directory/src/index.js

```diff
@@ -0,0 +1,106 @@
+"use strict";
```

Comment:
do you need this `index.js` file? that's the one executed by github job?

Reply:
no, this is the intermediary file : 
`src/index.ts` -> `src/index.js` -> `dist/index.js`
I'll update the .gitignore